eos-copy-host-mali-driver

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh -ex
+#
+# eos-copy-host-mali-driver: Copy the host's GL drivers into a flatpak local
+# extension directory.
+#
+# Copyright (C) 2017 Endless Mobile, Inc.
+# Authors:
+#  Philip Chimento <philip@endlessm.com>
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+EXTENSION_REF=org.freedesktop.Platform.GL.host/arm/1.6
+MALI_DIR=/usr/lib/arm-linux-gnueabihf/mali400-egl-x11
+
+DEST_DIR="/var/lib/flatpak/extension/$EXTENSION_REF"
+mkdir -p "$DEST_DIR"
+ln -s "$MALI_DIR" "$DEST_DIR/lib"
```

Comment:
As mentioned on Slack, what happens is flatpak binds DEST_DIR directly to /usr/lib/GL/host inside the flatpak, and sets the path to /usr/lib/GL/host/lib. So we need to create the parent of DEST_DIR, have DEST_DIR itself be the link to MALI_DIR, and the link in the ostree from lib -> . will put the libfoo*.so onto the library path.

As you're revising this PR, can I be a pain and have you rename all of this to eos-link-host-gl-driver because I expect @dsd will add nvidia to this mechanism for amd64 during 3.3. :)

Reply:
Actually, bikeshedding myself, host-mali-driver is OK. just link is better than copy. :)